lib/modules/hosts/imgur.js

```diff
@@ -77,6 +76,7 @@ const imgur = {
 			headers: {
 				Authorization: `Client-ID ${this.apiID}`,
 			},
+			cacheFor: DAY,
```

Comment:
Not sure if this is too long. Maybe `endpoint.startsWith('album/') ? HOUR : DAY` in case album captions are updated?